### PR TITLE
Resolve some codefactor issues

### PR DESCRIPTION
**Pull request by awaelchli**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?   
- [ ] Did you write any new necessary tests?  

## What does this PR do?
Fixes issues that codefactor is complaining about. See all issues [here](https://www.codefactor.io/repository/github/borda/pytorch-lightning/issues). 
I only looked at the easy ones and did not touch anything complex.
If you are not happy with some of the changes, let me know and I will revert these.

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by awaelchli** [comment]
GPU tests also pass, except amp which I don't have and ddp2 which needs SLURM and I don't have that.

File: tests/test_restore_models.py [hunk 1]
```diff
@@ -58,9 +58,9 @@ def test_running_test_pretrained_model_ddp(tmpdir):
 
 
 def test_running_test_pretrained_model(tmpdir):
+    """Verify test() on pretrained model"""
```
Comment (Borda) [inline tests/test_restore_models.py]: with the docs, a sentence should end with "."

**Comment by Borda** [comment]
@awaelchli could you pls rebase master...

**Comment by Borda** [comment]
@awaelchli btw, you may check also the codacy issues in another PR...
https://app.codacy.com/manual/Borda/pytorch-lightning/dashboard

**Comment by awaelchli** [comment]
yes I may take a look at these later. 
Also, do you know if it is possible to see codefactor/codacy results on a pull request to see if they are resolved?

**Comment by Borda** [comment]
I d love to see it too... I have asked @williamFalcon for adding codefactor here, creating PTL account because actually there is just my fork...

**Comment by williamFalcon** [comment]
@awaelchli rebase?

**Comment by awaelchli** [comment]
@williamFalcon rebased

**Comment by Borda** [comment]
@williamFalcon could you create PyTorch-Lightning account on codefactor.io? but it seems that it allows login with a GH account... or we may switch to Codacy.com which supports organisations...

**Pull request merged**